.gitignore

```diff
@@ -26,10 +26,11 @@ testbin/*
 *~
 /kuadrant-operator
 tmp
+topology.dot
```

Comment:
Probably no longer needed since the topology is saved to a ConfigMap?

Reply:
sure can do.